django_extensions/db/fields/encrypted.py

```diff
@@ -23,30 +23,11 @@ class BaseEncryptedField(models.Field):
     prefix = 'enc_str:::'
```

Comment:
Should we also use a different prefix ? such that we can detect old keyczar values ?